### DISCUSSION TITLE
Multiple Selection Widget

### DISCUSSION THREAD

**Discussion by gregcowell** [beeware/toga#1556] [Q&A] [answered]
Does Toga have a multiple selection widget - similar to the selection widget but allowing the user to select multiple items ?

**Comment by freakboy3742** [comment, marked as answer]
There's no "Multiselect" widget at present. You have the option of falling back to platform native APIs, or you might be able to cobble something together using a Table.

If you're interested in making a contribution, adding a multiselect widget should be relatively straightforward - the desktop platforms all have fairly direct mappings of a Multiselect widget.

**Reply by gregcowell** [reply]
Thanks, I am interested in making a contribution and I will give a Multiselect widget a try.

**Reply by freakboy3742** [reply]
Awesome - if you need any pointers or tips, let us know; but if you start with the existing Selection widget, and look to a widget like MultilineTextInput for details on how to deal with scrolling and widget sizing, you should be well on your way.
